### PR TITLE
Update spec.md

### PR DESCRIPTION
**Pull request by sebpuetz**

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -118,3 +142,16 @@ The chunk format is as follows:
 - Subquantizers: `quantized_len` x (`reconstructed_len` / `quantized_len`) x `sizeof(quantized_type)`
 - Norms: `matrix_rows` x `sizeof(reconstructed_type)`
 - Quantized embedding matrix: `matrix_rows` x `quantized_len` x `sizeof(reconstructed_type)`
+
+### Metadata
+
+- Chunk identifier: 5
+- UTF8 encoded Metadata: `chunk_data_len` times `u8`
```
Comment (danieldk) [inline spec.md]: Maybe add that the metadata is expected to be in TOML format.

**Review by danieldk — APPROVED** [review]
Thanks for adding these, small suggested addition.

File: spec.md (hunk 1 above)
Comment (sebpuetz) [reply spec.md]: done

**Comment by sebpuetz** [comment]
We should probably go through the embeddings at http://www.sfs.uni-tuebingen.de/a3-public-data/finalfusion/, sort out what we don't to publish on the website and list the ones that we want to distribute!

**Pull request merged**